ProjectDplusDsTree.py

```diff
@@ -146,16 +146,18 @@
     # compute pt weights
     if args.ptweights:
         ptWeights = uproot.open(args.ptweights[0])[args.ptweights[1]]
-        ptCentW = [(ptWeights.edges[iBin]+ptWeights.edges[iBin+1])/2 for iBin in range(len(ptWeights.edges)-1)]
+        bins = ptWeights.axis(0).edges()
+        ptCentW = [(bins[iBin]+bins[iBin+1])/2 for iBin in range(len(bins)-1)]
         sPtWeights = InterpolatedUnivariateSpline(ptCentW, ptWeights.values)
         dataFramePrompt['pt_weights'] = ApplySplineFuncToColumn(dataFramePrompt, 'pt_cand', sPtWeights, 0, 50)
         if not args.ptweightsB:
             dataFrameFD['pt_weights'] = ApplySplineFuncToColumn(dataFrameFD, 'pt_cand', sPtWeights, 0, 50)
             sPtWeightsDfromB = sPtWeights
 
     if args.ptweightsB:
-        ptWeightsB = uproot.open(args.ptweights[0])[args.ptweights[1]]
```

Comment:
@fgrosa Could this bug affect the pp results of the paper?

Reply:
CIao Fabio, I am not sure that I got which is the bug. Is the missing parenthesis?

Reply:
`args.ptweights` instead of `args.ptweightsB`

Reply:
Right, I didn't notice it, thanks for spotting it! In pp the pT weights should be very similar between prompt and FD, so it should not make any difference, but we better check